src/type/scalars.js

```diff
@@ -99,7 +112,7 @@ export const GraphQLFloat = new GraphQLScalarType({
   },
 });
 
-function coerceString(value: mixed): string {
+function serializeString(value: mixed): string {
```

Comment:
Can we make it strict similar to `serializeID` so it would expect only string and object with either `valueOf` or `toString`?
Or aleast throw on `serializeString({})` instead of returning `[object Object]`?

Reply:
Yeah, I agree the [object Object] scenario is useless - it should only string coerce if there's a toString implementation worth calling